File: No te limites/No te limites/Page.swift
//
//  Page.swift
//  No te limites
//
//  Created by Griselda Epifanio on 12/8/16.
//  Copyright © 2016 Jose Epifanio. All rights reserved.
//

import Foundation

class Page
{
    private var id:String = ""
    private var url:String = ""
    
    //Seters
    public func setId(id: String) {
        self.id = id
    }
    
    public func setUrl(url: String) {
        self.url = url
    }
    
    //Geters
    public func getId()-> String {
        return self.id
    }
    
    public func getUrl()-> String {
        return self.url
    }
}

File: Android/Notelimites/app/src/main/java/com/burocreativo/notelimites/io/models/Page.java
package com.burocreativo.notelimites.io.models;

/**
 * Created by Juan C. Acosta on 9/4/2016.
 * juancacosta25@gmail.com.com
 */
public class Page {
    private String id;
    private String url;

    public Page() {
    }

    public Page(String id, String url) {
        this.id = id;
        this.url = url;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}
